docs/releases/5.1.txt

```diff
@@ -63,6 +63,12 @@ Minor features
   accessibility of the
   :class:`~django.contrib.auth.forms.UserChangeForm`.
 
+* The :class:`~django.contrib.auth.middleware.LoginRequiredMiddleware` is now
+  available. It forces all views to require login by default. The
+  :class:`~django.contrib.auth.mixins.LoginNotRequiredMixin` mixin and
+  :func:`~django.contrib.auth.decorators.login_not_required` decorator have
+  also been added.
```

Comment:
@nessita do you think this should be a headline feature?

Reply:
Yes, I think this would be a nice adding to the `What's new in Django 5.1` section, good thinking Sarah! Thank you for proposing it.